### DISCUSSION TITLE
Low Quality (Python Bindings, LLAMA 13B Q4)

### DISCUSSION THREAD

**Discussion by R4mboX** [ggml-org/llama.cpp#978] [Q&A]
Hi There.

I'm using LLaMA 13B Q4_0 with the Python bindings in CPU Mode. 
I don't manage to get out good responses like I'm used to when working with the text-generation-webui.
I think I'm doing something wrong. 

Thats how I Instantiate the Model:
llm = Llama(model_path="./llama.cpp/models/13B/ggml-model-q4_0.bin", seed = 0, n_ctx = 1200)

And thats how I try to get a response:
output = llm(prompt, max_tokens=64, stop=[Human_Name + ":", "\n"], echo=True)

Technically everything works, but the quality of the response I get is quite off, and no where near I want it to be. 
It should be like in chat-mode. Mostly I get out of context responses, sometimes empty responses and sometimes gibberish.

Here's the Full Prompt I use in output = llm():
![image](https://user-images.githubusercontent.com/55106780/232132340-b6c441ec-38f7-4048-bf62-0d780164a87f.png)

Can you help me?

Some more Info about the Model:
![image](https://user-images.githubusercontent.com/55106780/232133265-2e778b7b-4e63-4a48-8ccc-4b0fc012602c.png)

Response Meta:
![image](https://user-images.githubusercontent.com/55106780/232133984-6e5d18ca-5b43-4c4a-99ca-779cffff1363.png)

**Comment by amirouche** [comment]
I had a similar xp.
